### PR TITLE
Add support for multi-core thread_id in filebeat postgresql module

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
Currently we only support postgresql log looks like:
> 2018-11-17 03:14:01 EST [835] LOG:  autovacuum launcher started

But this will fail when parsing postgresql log messages from multi-core machines which looks like:
> 2018-11-17 03:14:01 EST [835-1] LOG:  autovacuum launcher started

This PR is to add optional core id after the original thread id to parse multi-core postgresql logs. Please see more details in https://github.com/elastic/beats/issues/9156

closes https://github.com/elastic/beats/issues/9156

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -83,6 +83,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 - Added `detect_null_bytes` selector to detect null bytes from a io.reader. {pull}9210[9210]
 - Added `syslog_host` variable to HAProxy module to allow syslog listener to bind to configured host. {pull}9366[9366]
 - Added support on Traefik for Common Log Format and Combined Log Format mixed which is the default Traefik format {issue}8015[8015] {issue}6111[6111] {pull}8768[8768].
+- Add support for multi-core thread_id in postgresql module {issue}9156[9156]
```
Comment (jsoriano) [inline CHANGELOG.asciidoc]: Add also a reference to this PR please

File: filebeat/module/postgresql/log/test/postgresql-9.6-multi-core.log [hunk 2]
```diff
@@ -0,0 +1,19 @@
+2018-11-17 13:14:01 CEST [12975-1] [unknown]@[unknown] LOG:  incomplete startup packet
```
Comment (jsoriano) [inline filebeat/module/postgresql/log/test/postgresql-9.6-multi-core.log]: Add the expected file for this one please, I [mentioned](https://github.com/elastic/beats/pull/9431/files#r240010463) to remove the other one that was not related to this change.

**Comment by kaiyan-sheng** [comment]
@jsoriano Thanks for your help. I just created an issue for the timestamp problem: https://github.com/elastic/beats/issues/9494

**Comment by kaiyan-sheng** [comment]
@ruflin Sounds good! I just updated the PR description. Thanks!

**Pull request merged**